Runtime/Scripts/TextureUtils.cs

```diff
@@ -0,0 +1,45 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+
+namespace GLTFast
+{
+    static class TextureUtils
+    {
+        internal static string ConvertTexture(string texturePath, ImageFormat format)
+        {
+            string extension = format == ImageFormat.Jpeg ? ".jpg" : ".png";
+            string temp = "Assets/tempTexture" + extension;
```

Comment:
possible to store it in `Temp`?

Reply:
I tried it previously, but the `AssetDatabase` API used for loading the texture doesn't work out of the `Assets` directory, it seems.

Reply:
okok, as long as we cleanup the file it's ok then!